marlowe-playground-client/src/MainFrame.purs

```diff
@@ -101,9 +97,8 @@ mkMainFrame ::
   MonadAsk (SPSettings_ SPParams_) m =>
   n (Component HTML HQuery Unit Message m)
 mkMainFrame = do
-  editorPreferences <- Editor.loadPreferences
   let
-    initialState = mkInitialState editorPreferences
+    initialState = mkInitialState
```

Comment:
I would delete the let binding and rename `mkInitialState` to just `initialState`.

A "make" prefix makes me think it's doing some work. ie. it's got an `m State` signature. This is just static data.

Reply:
legacy rubbish.